src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+
```

Comment:
The `doWork` method should print a message indicating that the Truck has started working. Please add a print statement here.